Automattic/wp-calypso
# Update/extend resurrection event duration

Related to #
pau2Xa-4GE-p2#comment-13567

## Proposed Changes
Extend the resurrection day limit to 373 days and make it a prop.
*

## Testing Instructions
Setup Tracks Vigilante to log events.

Set up calypso locally.

Switch to this branch.

in [/client/lib/analytics/track-resurrections/index.jsx.](https://github.com/Automattic/wp-calypso/blob/update/extend-resurrection-event-duration/client/lib/analytics/track-resurrections/index.jsx#L7) change this line of code
`const RESURRECTION_DAY_LIMIT = 373;`
to
`const RESURRECTION_DAY_LIMIT = 0.0035; // This triggers a resurrection if a user hasn't logged in for 5 minutes.`

The change above will ensure that we don't have to wait for one year but 5 minutes to see a resurrection event.

Log out of Calypso.

Wait for 5 minutes or more.

Log back in and immediately visit /home/${ primarySiteSlug }.

You should see a resurrection event logged in Vigilante with a last_seen prop that is a timestamp in seconds.
And a new prop, "day_limit" which has a value of 0.0035


*

## Pre-merge Checklist
